### PR TITLE
Ensure new sites are inactive on create

### PR DESCRIPTION
**Pull request by nspool**

### CODE REVIEW

**Comment by nspool** [comment]
Updated to make the column not null and false by default.

File: api/src/main/resources/sql/migration.sql [hunk 1]
```diff
@@ -551,7 +551,7 @@ CREATE TABLE nrmn.site_ref (
     wave_exposure integer,
     slope integer,
     site_attribute jsonb,
-    is_active boolean
+    is_active boolean NOT NULL
```
Comment (jonescc) [inline api/src/main/resources/sql/migration.sql:554]: This also needs to be done in https://github.com/aodn/NRMN  - contains the master ddl for core tables.  But can address that separately.

**Review by jonescc — APPROVED** [review]

File: api/src/main/resources/sql/migration.sql (hunk 1 above)
Comment (jonescc) [reply api/src/main/resources/sql/migration.sql:554]: https://github.com/aodn/NRMN/blob/master/db-migrate/ddl/CreateTables.sql#L42

Comment (bpasquer) [reply api/src/main/resources/sql/migration.sql:554]: > This also needs to be done in https://github.com/aodn/NRMN - contains the master ddl for core tables. But can address that separately.

Yes I've prepared a PR  to set all `is_active` attributes NOT NULL in the migration code

**Pull request merged**